apps/web/src/app/_components/footer.tsx

```diff
@@ -18,9 +19,17 @@ export default function Footer() {
         />
         <h2 className="text-xl font-semibold">Checkout cooper on GitHub</h2>
       </Link>
-      <a href="https://clearbit.com" target="_blank" className="italic">
-        Logos provided by Clearbit
-      </a>
+      <div className="flex flex-col items-center md:flex-row md:gap-6">
+        <Link href="____" target="_blank" className="flex items-center gap-2">
```

Comment:
Let's use this link

https://docs.google.com/forms/d/e/1FAIpQLSf-ZvpyBawT5LlMho7X4oNZ0Z_1M-o6cXLJjB5uJiNAvvnkfw/viewform?usp=dialog